manifests/cf-manifest/operations.d/710-rds-broker.yml

```diff
@@ -3,9 +3,9 @@
   path: /releases/-
   value:
     name: rds-broker
-    version: 1.8.0
-    url: https://s3-eu-west-1.amazonaws.com/gds-paas-build-releases/rds-broker-1.8.0.tgz
-    sha1: 9264469b832061f15479df96440ee6ad1280692b
+    version: 0.0.1629903410
```

Comment:
Just a reminder to update this before release, as it may be hidden in the commit messages.